install_sct

```diff
@@ -636,16 +656,20 @@ python -c 'import spinalcordtoolbox.deepseg.models; spinalcordtoolbox.deepseg.mo
 
 # In case of previous SCT installation (4.0.0-beta.1 or before), remove sct_env declaration in bashrc
 print info "In case an old version SCT is already installed (4.0.0-beta.1 or before), remove 'sct_env' declaration in RC file"
-if [[ $OS == "osx" ]]; then
-  sed -ie '/sct_env/ s/^#*/#/' $RC_FILE_PATH
+if [[ -e "$RC_FILE_PATH" && $OS == "osx" ]]; then
+  sed -ie '/sct_env/ s/^#*/#/' "$RC_FILE_PATH"
 else
-  sed -e '/sct_env/ s/^#*/#/' -i $RC_FILE_PATH
+  sed -e '/sct_env/ s/^#*/#/' -i "$RC_FILE_PATH"
 fi
```

Comment:
The `-e` existence check was added to the `osx` case but not the else. Would nested ifs be better? e.g.

```bash
if [[ -e "$RC_FILE_PATH" ]]; then
  if [[ $OS == "osx" ]]; then
    sed -ie '/sct_env/ s/^#*/#/' "$RC_FILE_PATH"
  else
    sed -e '/sct_env/ s/^#*/#/' -i "$RC_FILE_PATH"
  fi
fi
```